tools/dynamic_framework_slicer/dynamic_framework_slicer.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash 
+set -eu
+
+BINARIES+=()
```

Comment:
shouldn't need `+=` here, just `=`, right?

Reply:
Yep that should be fine.